src/SwedbankPay.Sdk/Payments/SwishPayments/Operations.cs

```diff
@@ -2,14 +2,14 @@
 using System.Collections.Generic;
 using System.Threading.Tasks;
 
-namespace SwedbankPay.Sdk.Payments.Swish
+namespace SwedbankPay.Sdk.Payments.SwishPayments
 {
     public class Operations : Dictionary<LinkRelation, HttpOperation>
     {
         public Func<Task<PaymentResponse>> Abort { get; internal set; }
         public Func<ReversalRequest, Task<ReversalResponse>> CreateReversal { get; internal set; }
```

Comment:
It's OK that the `const string` is named `CreateReversal` to reflect the underlying `rel` value, but the logical function performed is `Reverse` so it makes more sense to expose it as `Reverse` in C#. The same goes for `CreateCapture` (should be `Capture`), `CreateCancel` (should be `Cancel`), etc.

```suggestion
        public Func<ReversalRequest, Task<ReversalResponse>> Reverse { get; internal set; }
```